test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -87,14 +87,8 @@ EXPLAIN WITH(arity, join_impls)
       (SELECT a, a+1 as b FROM y))
 WHERE b = 3 AND c = 3
 ----
-Explained Query:
-  Filter (#1 = 3) AND (#2 = 3) // { arity: 3 }
-    Map ((#0 + 1), (#1 + 1)) // { arity: 3 }
-      ReadStorage materialize.public.y // { arity: 1 }
-
-Source materialize.public.y
-  filter=((3 = #1) AND (3 = (#1 + 1)))
-  map=((#0 + 1))
+Explained Query (fast path):
+  Constant <empty>
```

Comment:
@ggevay This query may want to be refreshed to test a new thing.

Reply:
Test modified, and will separately think about how to include several tests that lock in this sort of behavior.